collections/_article/dev-snapshot-godot-4-5-dev-2.md

```diff
@@ -0,0 +1,114 @@
+
+Steps have been taken during this initial 4.5 period to ensure that navigation logic is more cleanly divided and organized, spearheaded by our navigation guru [smix8](https://github.com/smix8), but none of this could've been possible without cleanly splitting the navigation server in twain. Longtime contributor [AThousandShips](https://github.com/AThousandShips) took the mantle on this project, and she successfully accomplished this goal with [GH-101504](https://github.com/godotengine/godot/pull/101504). From now on, users will be able to selectively enable/disable the navigation modules for 2D and/or 3D, instead of being forced into an all-or-nothing ultimatum.
+
+### Reorganized shader editor UI
+
+The editor experience for shaders and visual shaders got some TLC in [GH-100287](https://github.com/godotengine/godot/pull/100287). Helmed by [Yuri Rubinsky](https://github.com/Chaosus), this PR provides several requested features and QOL updates to our shading friends. Below is a preview image taken directly from the PR in question, where you'll find more information on what to expect when you get your hands on it this snapshot.
+
+![Updated the shader editor UI](/storage/blog/dev-snapshot-godot-4-5-dev-2/shader-editor.webp)
+
+### Changing editor language without restart
+
+Godot has built-in support for handling multiple languages, both for the games you create and the editor itself. However, in contrast to games built with the engine, it wasn't possible to change the current language on-the-fly within the editor itself. Our editor expert [Tomasz Chabora](https://github.com/KoBeWi) addressed this issue in [GH-102562](https://github.com/godotengine/godot/pull/102562), ensuring that users can swap to their preferred language within the same editor session! While this is somewhat niche in practice, our stance on convenience and accessibility is one that we take very seriously, and a seamless/streamlined editor experience is exactly the sort of thing we want as many users as possible to enjoy.
+
+### Wayland: Native Sub-Window Support
+
+Thanks to the tireless efforts of [Riteo](https://github.com/Riteo), the X11 alternative [Wayland](https://wayland.freedesktop.org/) has been given first-class treatment on Godot. It's been a long road to stand as an equal to such a dominant display server protocol, but it's getting closer with every passing PR; though some have been hesitant to make the change for one reason: lack of native sub-windows. Taking on this hurdle for parity was no small feat, especially for a comparatively immature backend for a comparatively immature protocol, but [GH-101774](https://github.com/godotengine/godot/pull/101774) made it happen all the same:
```

Comment:
The "immature" comments are maybe a tad much, we don't want to start an open flamewar with the Wayland fans :D

Reply:
Instead, what could be worth mentioning is teasing that this is a pre-requisite to supporting the embedded game window on Wayland too (though this may also expose us to macOS users complain about being left out :P).